locust/contrib/postgres.py

```diff
@@ -0,0 +1,52 @@
+from locust import TaskSet, User, events
+
+import time
+
+import psycopg
+
+
+def create_conn(conn_string):
+    return psycopg.connect(conn_string)
+
+
+class PostgresClient:
+    def __init__(self, conn_string):
+        self.conn_string = conn_string
+        self.connection = create_conn(conn_string)
+
+    def execute_query(self, query):
+        start_time = time.time()
+        try:
+            cursor = self.connection.cursor()
```

Comment:
psycopg3 can do connection.execute(), no need to have the intermediate step of creating a cursor.